unstructured/ingest/evaluate.py

```diff
@@ -69,88 +44,29 @@ def main():
     help="A tuple of weights to the Levenshtein distance calculation. \
         See text_extraction.py/calculate_edit_distance for more details.",
 )
-def measure_text_edit_distance(
+def measure_text_edit_distance_command(
     output_dir: str,
-    output_list: Optional[List[str]],
     source_dir: str,
+    output_list: Optional[List[str]],
     source_list: Optional[List[str]],
```

Comment:
```suggestion
    output_list: Optional[List[str]] = None,
    source_list: Optional[List[str]] = None,
```

Reply:
Same as other function, defaults are handled by click. Let me know if I should change it anyway.